package.json

```
@@ -75,16 +87,14 @@
     "@types/express": "^4.17.13",
     "@types/express-fileupload": "^1.5.0",
     "@types/http-errors": "^2.0.2",
-    "@types/jest": "*",
     "@types/node": "*",
     "compression": "^1.7.4",
     "express": "^4.19.2",
     "express-fileupload": "^1.5.0",
+    "eslint": "^8.0.0 || ^9.0.0",
```

```suggestion
    "eslint": "^8.57.0 || ^9.0.0",
```

the `0` one didn't have flat config by default